.github/workflows/release-drafter.yml

```diff
@@ -4,17 +4,21 @@ on:
   push:
     branches:
       - main
+      - test/angular-v*
```

Comment:
We can remove this line because in the ideal situation we don't need these branches. And if we create them, we will modify the condition for the workflow in a branch.